packages/ckeditor5-image/src/image/imageinline.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module image/image/imageinline
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+
+import { toImageWidget, createImageViewElement, getImageTypeMatcher } from './utils';
+
+/**
+ * The image engine plugin.
+ *
+ * It registers:
+ *
+ * * `<image>` as a block element in the document schema, and allows `alt`, `src` and `srcset` attributes.
+ * * converters for editing and data pipelines.
+ * * `'imageInsert'` command.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class ImageInline extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'ImageInline';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const schema = editor.model.schema;
+		const t = editor.t;
+		const conversion = editor.conversion;
+
+		schema.register( 'imageInline', {
+			isObject: true,
+			isInline: true,
+			allowWhere: '$text',
+			allowAttributes: [ 'alt', 'src', 'srcset' ]
```

Comment:
Converters for those attributes should be registered in this file.

Reply:
OTOH they are identical (converting only `alt` and `srcset` attributes on an `img` element so maybe there should stay in `ImageEditing` and in this file, it should be mentioned where they can be found.